frontend/src/views/applications/OperatingCostsView.vue

```diff
@@ -190,7 +190,11 @@ export default {
           this.setSuccessAlert('Application saved successfully')
         }
       } catch (error) {
-        this.setFailureAlert('Failed to save your application', error)
+        if (error?.response?.data?.status === 422) {
+          this.setFailureAlert('Failed to save your application supporting documents failed virus scan', error)
```

Comment:
This is a little verbose for the user. Maybe something like "Supporting documents not saved. Virus scan failed."